extensions-core/google-extensions/src/test/java/org/apache/druid/storage/google/GoogleDataSegmentKillerTest.java

```diff
@@ -99,16 +95,11 @@ public void killTest() throws SegmentLoadingException, IOException
     verifyAll();
   }
 
-  @Test(expected = SegmentLoadingException.class)
-  public void killWithErrorTest() throws SegmentLoadingException, IOException
+  @Test(expected = RE.class)
```

Comment:
Junit 5.x doesn't support this style of expected exceptions. In light of that, it'd be best to migrate to `Assert.assertThrows` in this file while we're modifying this code. This could also come in a follow up patch.

Reply:
Modified